src/app.js

```diff
@@ -22,24 +22,24 @@ dotenv.config();
 const connectedClients = {};
 io.use(async (socket, next) => {
   const { token } = socket.handshake.query;
-  if (token) {
+  try {
     const decoded = decodeToken(token, process.env.SECRETKEY);
     const userData = decoded;
-    if (!userData.error) {
-      const clientKey = Number.parseInt(userData.id, 10);
-      connectedClients[clientKey] = connectedClients[clientKey] || [];
-      connectedClients[clientKey].push(socket.id);
-    }
+    const clientKey = Number.parseInt(userData.id, 10);
+    connectedClients[clientKey] = connectedClients[clientKey] || [];
+    connectedClients[clientKey].push(socket.id);
+    next();
+  } catch (error) {
+    console.log('Invalid token provided');
```

Comment:
hey @Kagorora , isn't this supposed to be returned as a response?

Reply:
@T2Wil  when returning error in the catch, I can't see that message that's why I have consoled this